debezium-core/src/test/java/io/debezium/time/TimeTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Debezium Authors.
+ * 
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.time;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+/**
+ * @author Randall Hauch
+ */
+@Ignore
```

Comment:
I think tests are good (i.e. the more the merrier) but these 2 just seem to print out stuff, without asserting anything. So in this form I'm not sure they make sense. Maybe with some asserts they would ?